crates/sui-framework/sources/coin.move

```diff
@@ -27,6 +31,25 @@ module sui::coin {
         balance: Balance<T>
     }
 
+    /// Each Coin<T> will have a unique instance of CoinMetadata<T> that
```

Comment:
It is only unique if you go through `create_currency`. Not sure how you want to document that. 

But if you make a coin via balance, it won't have a CoinMetadata. Which is fine! But not clear from this documentation